lib/lobbyist/v2/customer_call.rb

```diff
@@ -1,24 +1,35 @@
 module Lobbyist
   module V2
+    class CustomerCall < Lobbyist::Base
```

Comment:
Oh, thanks, I didn't change the base class name too.
I just moved CustomerCall to V2::CustomerCall, but there was already class V2::CustomerCall so I moved it to V2::CompanyCustomerCall.